stackhpc_ipa_hardware_managers/system_bios.py

```diff
@@ -66,9 +66,11 @@ def _get_expected_property(node, property):
     try:
         expected_property = node['extra']['system_vendor'][property]
     except KeyError as e:
-        raise errors.CleaningError("Expected property not found. For "
-                                   "cleaning to pass you must set the "
-                                   "property name: {}".format(e))
+        raise errors.CleaningError(
+            "Expected property '{0}' not found. For cleaning to proceed "
+            "you must set the property '{1}', for example: "
```

Comment:
I think this is still confusing because it's not a property under node.properties. Given that we know the full path to the field, can we provide it? E.g. 'system_vendor/{} in the node's extra field'?

Reply:
np, i've added it to the text before the example.